java/semantickernel-api/src/main/java/com/microsoft/semantickernel/util/AzureOpenAISettings.java

```diff
@@ -5,6 +5,7 @@
 
 import javax.annotation.Nullable;
```

Comment:
Should this (and OpenAISettings) be defined here or moved to the OpenAI connector?